### DISCUSSION TITLE
Typescript Intellisense Not Woking for Fluent API

### DISCUSSION THREAD

**Discussion by niranjangajare** [prisma/prisma#11292] [Q&A]
is it just me or anyone else not getting the typescript intellisense for Fluent API?

I am using Latest Version of Prisma with Next.js setup.

**Comment by austincrim** [comment]
Hey @niranjangajare 👋 ,

I am not aware of any issues related to the TypeScript and the fluent API. What kind of behavior are you seeing that is breaking with expectation? An example with screenshots or screen recording would be great.

**Reply by niranjangajare** [reply]
[The Fluent API Docs](https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries#fluent-api) Section doesn't show any example with Fluent API function call usage with arguments.

I Found out about the Arguments by going through the Type Definitions.
but for some reasons VS Code is unable get intellisense from the Types.
Here, the resolver functions runs as expected and gets correct result, the only problem is that the arguments `{ skip, take }` is not intellisensed regardless of the Type provided by `@prisma/client`

```typescript
resolve: async ({ sensorId }, { skip, take }, { prisma }) => {
	const data = await prisma.tempSensor
		.findUnique({ where: { sensorId } })
		.tempLog({ skip, take })
	return data
}
```

```typescript
// Type Generated in @prisma/client
tempLog<T extends TempLogFindManyArgs = {}>(args?: Subset<T, TempLogFindManyArgs>): 
       CheckSelect<T, PrismaPromise<Array<TempLog>>, PrismaPromise<Array<TempLogGetPayload<T>>>>;
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
